ogr/ogrsf_frmts/sqlite/ogr_sqlite.h

```diff
@@ -673,6 +676,12 @@ class OGRSQLiteDataSource final : public OGRSQLiteBaseDataSource
     int                 GetUndefinedSRID() const { return m_nUndefinedSRID; }
     bool                HasGeometryColumns() const { return m_bHaveGeometryColumns; }
 
+    void                LoadRelationships() const;
```

Comment:
I'm wondering if that shouldn't be moved to the OGRSQLiteBaseDataSource class, which is an ancestor for GPKG, and GPKG fall back to that method when gpkgext_relations extension isn't used (there's nothing AFAIK in the base GPKG spec that prevents using sqlite3 foreign keys)

Reply:
Good idea! I've done that now